releasenotes/notes/dogstatsd-perf-cpu-ab84ee8825b4b3cb.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The performance of the Agent under DogStatsD load has been improved.
```

Comment:
Do we want to develop a bit more for customers ? Or maybe add a link to this PR ?